File: Simulation/3324.Find the Sequence of Strings Appeared on the Screen/Solution.py
class Solution:
    def stringSequence(self, target: str) -> List[str]:
        ans = []
        s = ""
        for t in target:
            v = 'a'
            ans.append(s[:] + v)
            while v != t:
                v = chr(ord(v) + 1)
                ans.append(s[:] + v)
            s += v
        return ans

File: Simulation/3324.Find the Sequence of Strings Appeared on the Screen/Solution.java
class Solution {
    public List<String> stringSequence(String target) {
        List<String> ans = new ArrayList<>();
        String s = "";
        for (char t : target.toCharArray()) {
            char v = 'a';
            ans.add(s + v);
            while (v != t) {
                v++;
                ans.add(s + v);
            }
            s += v;
        }
        return ans;
    }
}